### DISCUSSION TITLE
Can object be combined with labelmap.txt

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#1435] [General]
There are lots of objects frigate ai can detect, for example i have person, car, bus, bicycle, dog, cat and so
On the frigate UI it shows the events for all the objects i have added.
wondering if its possible to group them together and name them custom like

vehicle=car,bus,bicycle
animals=dog,cat 

in this way on the events i would only see vehicle and animals.
I checked out labelmap.txt and it seem we can rename the object name but nothing about grouping

any idea?

thanks for the great program. I love it.

**Comment by blakeblackshear** [comment]
https://blakeblackshear.github.io/frigate/configuration/objects#customizing-the-labelmap

**Comment by blakeblackshear** [comment]
Just rename multiple to the same value to group.

**Reply by balajeek** [reply]
I tried like this:  in labelmap.txt

0  person
1  vehicle
2  vehicle
3  vehicle
5  vehicle
7  vehicle
15  animal
16  animal
17  animal
22  animal

Then in the config.yml

    objects:
      track:
        - person
        - animal
        - vehicle

After reload of frigate, the ui refreshed to those three groups i mentioned, but only person seem to be working (becaz its just only one of them), animal and vehicle is empty.

Am I doing something wrong?

**Reply by blakeblackshear** [reply]
You can't delete lines

**Reply by balajeek** [reply]
you mean i have to keep the labelmap.txt with full list and modify the label that i want !

**Reply by blakeblackshear** [reply]
Yes. Also, please clarify what you mean by "animal and vehicle is empty".

**Reply by balajeek** [reply]
what I meant empty is after i made the change then see on the frigate UI events, I see three tracked objects, (person,animal,vehicle) and i only see snapshot/clips in person, but nothing detected on animal or vehicle.

I will try redo the labelmap.txt with full list as you suggested.
thanks again for the quick response, I really appreciate.

**Reply by blakeblackshear** [reply]
It won't change old events. You will need to wait for a new vehicle or animal for it to show up.

**Reply by balajeek** [reply]
Yes, I can confirm. putting the labelmap.txt as it is with full list, and modify the labels (like changing dog and cat to animals), reload.  same result, only person is detected not animals

I will wait for few min to see if car object picked up, becauze there are two cars outside now.
also on person object it did show all the previous events from yesterday and today, its only the modified label for animal and vehicle is not detected.

will keep you posted in few min.
thanks.

**Comment by balajeek** [comment]
You are right, few hours later i checked and the new detected objects are in the respective folder as I named them.
cool it took a great clip of bird in see in the group.
Awesome, and thanks for your support and great program.
